### PR TITLE
Prayer fix fix

### PR DESCRIPTION
**Pull request by Just-a-Unity-Dev**
significant proof I am a good developer

### CODE REVIEW

**Comment by Just-a-Unity-Dev** [comment]
did i just typo at the beginning

**Comment by Just-a-Unity-Dev** [comment]
i hate github webedit

**Comment by Just-a-Unity-Dev** [comment]
i hate github webedit

File: Content.Server/Prayer/PrayerSystem.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-p﻿using Content.Server.Administration;
```
Comment (Veritius) [inline Content.Server/Prayer/PrayerSystem.cs]: pusing

**Comment by moonheart08** [comment]
Did you actually test this?

**Comment by Just-a-Unity-Dev** [comment]
> Did you actually test this? (No, go test your shit)

i should stop making PRs in a middle of a lecture

**Comment by Just-a-Unity-Dev** [comment]
ill just close this PR and fix it in another one

**Comment by Just-a-Unity-Dev** [comment]
ill just close this PR and fix it in another one

**Pull request closed without merging**